book-example/src/format/config.md

```diff
@@ -14,6 +14,10 @@ description = "The example book covers examples."
 build-dir = "my-example-book"
 create-missing = false
 
+[preprocess]
+index  = true
```

Comment:
I don't think this is correct. Each preprocessor gets its own table and you can enable/disable inserting the default preprocessors by setting `build.use-default-preprocessors` to `true`.